### DISCUSSION TITLE
TP3 - Copiar cadenas seguras

### DISCUSSION THREAD

**Discussion by Nikzza** [INGCOM-UNRN-P1/cursada-2023#101] [Preguntas y respuestas] [answered]
tengo una consulta, en el tp3.2 dice que deje con un \0 en la cadena destino, no seria en la cadena original?

**Comment by martinvilu** [comment]
¿Y cuál sería la consigna?

**Reply by Nikzza** [reply]
Implementar una función que copie una cadena en otra y deje solo un \0 en la cadena destino

La función debe retornar el largo de la cadena destino o el código de error correspondiente.
Yo creo que el \0 es para la cadena anterior no para la de destino

**Reply by martinvilu** [reply, marked as answer]
Es para donde haces la modificación, dejar solo un /0

**Reply by Nikzza** [reply]
esta bien, y tenia otra consulta. Hace falta mostrar como se queda insertada una cadena en otra en el tp3.3?, el ejercicio dice
"Implementar una función que inserte una cadena en otra luego de la posición indicada de manera segura."

**Reply by martinvilu** [reply]
En `main` como todas las otras funciones.
